src/components/il-tabs/il-tabs.component.js

```diff
@@ -94,6 +117,7 @@ class TabComponent extends LitElement {
   }
 
   render() {
+    this.setTabs();
```

Comment:
I'm not sure we want to do this on every render. Under what circumstances do we *need* to reformat the tabs?

Reply:
I think we do need to do this on every render to manage the case where things get added or removed.

Reply:
Did you try the mutation events for that?

Reply:
I tried, but I may have been doing it wrong. I created an issue for this